types/jest-retries/index.d.ts

```diff
@@ -0,0 +1,9 @@
+// Type definitions for jest-retries 1.0
+// Project: https://github.com/bluzi/jest-retries#readme
+// Definitions by: Florent Jaby <https://github.com/Floby>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+type DoneCallback = (...args: any[]) => any;
+type ProvidesCallback = (done?: DoneCallback) => any;
+declare function JestRetries(desc: string, retries: number, fn?: ProvidesCallback, timeout?: number): void;
+export = JestRetries;
```

Comment:
```suggestion
declare function retry(desc: string, retries: number, fn?: ProvidesCallback, timeout?: number): void;
export = retry;
```
I'd say 'Camel'-case function notation is non-standard in JS, why just not use source name?:
https://github.com/bluzi/jest-retries/blob/master/src/retry.js#L14

Reply:
Good catch on the polyformism, I didn't know it did this. I wrote the types from the readme doc rather than the source, which explains some naming oddities (others are just that I'm new to this).

timeout is a an argument which the original jest `test` function has for the maximum duration of a given test. but it seems this module doesn't actually forwards it.

Reply:
This comment hasn’t been resolved—the function signature still looks wrong.